File: src/java/main/commands/RichJson_copy.java
package commands;

import core.RichJsonCommand;
import core.RichJsonCommandHolder;
import core.RichJsonContext;
import helper.RichJsonHelper;
import core.RichJsonParser;

public class RichJson_copy implements RichJsonCommand {
    @Override
    public Object execute(RichJsonParser parser, RichJsonContext context) throws Exception {
        Object referencedValue = RichJsonCommandHolder.executeCommand("ref", parser, context);
        return RichJsonHelper.cloneObject(referencedValue);
    }
}

File: src/c#/main/commands/RichJson_copy.cs
using System;
using main.core;
using main.helper;

namespace main.commands
{
    public class RichJson_copy
    {
        public object Execute(RichJsonParser parser, RichJsonContext context)
        {
            object referencedValue = RichJsonCommandHolder.ExecuteCommand("ref", parser, context);
            return RichJsonHelper.CloneObject(referencedValue);
        }
    }
}